modmesh/pilot/_svg.py

```diff
@@ -148,191 +148,272 @@ def calc_vertices(self):
         current_pos = start_pos
         last_control = None
         last_cmd = None
-
         for idx, (cmd, coords) in enumerate(commands):
             i = 0
             if cmd in ('M', 'm'):
-                # move to (x, y):
-                #   M x y
-                #   m dx dy (relative position to current position)
-                x, y = coords[i], coords[i+1]
+                # Move position:
+                #   command: [M|m] (relative position in lowercase command)
+                #   parameter: (dx, dy)+
+                dx, dy = coords[i:i+2]
+                x_cur, y_cur = current_pos[0], current_pos[1]
 
                 if cmd == 'M':
-                    start_pos[0] = x
-                    start_pos[1] = y
-                else:  # cmd is 'm', relative position
-                    start_pos[0] = (current_pos[0] + x)
-                    start_pos[1] = (current_pos[1] + y)
+                    start_pos[0] = dx
+                    start_pos[1] = dy
+                else:
+                    start_pos[0] = (x_cur + dx)
+                    start_pos[1] = (y_cur + dy)
 
                 current_pos = start_pos
                 i += 2
+
+                # implicit line-to command
+                while i + 1 < len(coords):
```

Comment:
As described in [moveTo command](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/d#moveto_path_commands):

> Any subsequent coordinate pair(s) are interpreted as parameter(s) for implicit absolute LineTo (L) command(s) 

It is required to add segments in `M` and `m` commands.

Reply:
Is it covered unit test?